src/test/java/TestBankAccountTransactionFee.java

```diff
@@ -14,7 +14,7 @@
         return Arrays.asList(new Object[][] {
                 {5.00, true},
                 {7.00, true},
-                {20.00, false},
+                {10.00, false},
```

Comment:
This is reverting the previous Derrick's modification. You have to change back to `20.00`.

See: https://cornerstone19jan.slack.com/archives/CF86RNG68/p1551384877004100